pom.xml

```diff
@@ -67,24 +67,24 @@ limitations under the License.
     <hbase.version.1>1.4.12</hbase.version.1>
     <hbase.version.2>2.2.3</hbase.version.2>
     <hbase.version>${hbase.version.1}</hbase.version>
-    <hbase1-hadoop.version>2.7.4</hbase1-hadoop.version>
+    <hbase1-hadoop.version>2.10.1</hbase1-hadoop.version>
     <hbase2-hadoop.version>2.8.5</hbase2-hadoop.version>
 
     <!-- testing dependency versions -->
     <commons-lang.version>2.6</commons-lang.version>
     <junit.version>4.13.1</junit.version>
     <hamcrest.version>1.3</hamcrest.version>
-    <mockito.version>3.6.0</mockito.version>
+    <mockito.version>3.7.7</mockito.version>
     <!-- TODO: check if commons-codec was transitively updated to 1.13 and okhttp was updated to 2.7.5 when upgrading-->
     <beam.version>2.24.0</beam.version>
     <!-- referred from bigtable-beam-import and bigtable-emulator -->
-    <guava.version>30.0-android</guava.version>
-    <beam-guava.version>20.0</beam-guava.version>
-    <beam-auto-value.version>1.7</beam-auto-value.version>
+    <guava.version>30.1-android</guava.version>
+    <gcs-guava.version>29.0-jre</gcs-guava.version>
+    <beam-auto-value.version>1.7.4</beam-auto-value.version>
     <beam-grpc.version>1.29.0</beam-grpc.version>
 
     <!-- Benchmarks related dependencies -->
-    <jmh.version>1.23</jmh.version>
+    <jmh.version>1.27</jmh.version>
```

Comment:
why all of the version upgrades?

Reply:
These upgrades were done as part of HBaseSnapshotImport work, I am bringing them as is. I don't know if the import job breaks if we downgrade them.